### ISSUE TITLE
demo

### ISSUE THREAD

**Issue by skacco** [oc-shopaholic/oc-orders-shopaholic-plugin#188]
demo credential (manager/manager) is not working 
https://demo.shopaholic.dev/backend/backend/auth/signin

**Comment by xApep** [comment]
it sux yeah :)

**Comment by daslicht** [comment]
still broken

**Comment by baberuth22** [comment]
What do you mean broken?   Create a proper login.   Dooesn't speak well for the project...

Please, use the following credentials user:manager,password:manager for logging to the [Dashboard](https://demo.shopaholic.dev/backend).

**Comment by xApep** [comment]
> Doesn't speak well for the project...
> 
> Please, use the following credentials user:manager,password:manager for logging to the [Dashboard](https://demo.shopaholic.dev/backend).

@baberuth22 did you even try with those credentials? Cause they do not work!

![image](https://github.com/oc-shopaholic/oc-orders-shopaholic-plugin/assets/1315340/bbeeacb8-05dc-4f0e-9e74-9d519ea30eae)

**Comment by daslicht** [comment]
Yeh if something starts like this I'd rather look for alternatives with better support.

**Comment by xApep** [comment]
> Yeh if something starts like this I'd rather look for alternatives with better support.

I agree, I was looking for e-commerce CMS, found October CMS, but gave up, since the "only" solid extension is dead. Clap, clap!
